packages/thirdweb/src/extensions/erc1155/drop1155.test.ts

```diff
@@ -45,12 +46,21 @@ describe.runIf(process.env.TW_SECRET_KEY)(
         chain: ANVIL_CHAIN,
         client: TEST_CLIENT,
         params: {
-          name: "Test DropERC1155",
-          contractURI: TEST_CONTRACT_URI,
+          name: "EditionDrop",
```

Comment:
Can You still pass a contract uri here? Was useful to skip the ipsy upload

Reply:
oops

Reply:
added back